### ISSUE TITLE
Request for Details and Code on Baselines

### ISSUE THREAD

**Issue by Loserm-Ricardo** [cpystan/Wsi-Caption#14]
Thank you for your remarkable contributions to the field of pathology report generation. Given the limited research in this area and the scarcity of available datasets and code, your work and the publicly released resources are of immense value to advancing studies in this domain.
I am currently attempting to reproduce the results from your paper. As mentioned, there are few available report generation models, which makes the baselines on pathology report generation described in Table 1 of your paper particularly interesting.
I would like to better understand **how these classic models were adapted for the pathology report generation task**. Could you kindly provide the code for the baselines used in your paper? Sharing these details would further propel research in this field and be greatly appreciated.
Thank you for your time and for sharing your excellent work with the community.

**Comment by Loserm-Ricardo** [comment]
My understanding is that these models were not specifically adapted for the task of pathology report generation. Instead, it seems that these are standard report generation models, where features extracted from thousands of patches are directly fed into the model.
Could you confirm if my understanding is correct? Any additional details or clarification would be greatly appreciated.

**Comment by cpystan** [comment]
Yes, the patch embeddings are directly fed to the classic models. The difference between our model and baselines is the encoder-decoder.

**Comment by SuWeiheng** [comment]
Encountered the same problem. I'm also confused by the classification model provided in the baseline folder. 
When checking files in the baseline folder, models for classification were found but not models for report generation like  Mem-Transformer, att-LSTM mentioned in the original paper.
It's much appreciated if you could provide the baseline used in the paper for further research. Thank you for your time and for sharing your excellent work with the community. @cpystan @Loserm-Ricardo

**Comment by SuWeiheng** [comment]
Encountered the same problem. I'm also confused by the classification model provided in the baseline folder.
When checking files in the baseline folder, models for classification were found but not models for report generation like **Mem-Transformer, att-LSTM** mentioned in the original paper.
Baselines for slide-level tasks were found but the **code for conducting slide-level tasks was not mentioned** in the readme file nor in any folders.
It's much appreciated if you could provide the baseline used in the paper for further research. Thank you again for your research. @cpystan

**Comment by SuWeiheng** [comment]
@cpystan

**Comment by cpystan** [comment]
> Encountered the same problem. I'm also confused by the classification model provided in the baseline folder. When checking files in the baseline folder, models for classification were found but not models for report generation like **Mem-Transformer, att-LSTM** mentioned in the original paper. Baselines for slide-level tasks were found but the **code for conducting slide-level tasks was not mentioned** in the readme file nor in any folders. It's much appreciated if you could provide the baseline used in the paper for further research. Thank you again for your research. @cpystan
In terms of the evaluation code, you can refer to https://github.com/cpystan/WSI-VQA/blob/master/downstream/show.ipynb. Given the text, we search the corresponding key word in the report and calculate the accuracy if correctly matched.

**Comment by cpystan** [comment]
> Encountered the same problem. I'm also confused by the classification model provided in the baseline folder. When checking files in the baseline folder, models for classification were found but not models for report generation like **Mem-Transformer, att-LSTM** mentioned in the original paper. Baselines for slide-level tasks were found but the **code for conducting slide-level tasks was not mentioned** in the readme file nor in any folders. It's much appreciated if you could provide the baseline used in the paper for further research. Thank you again for your research. @cpystan

In terms of the evaluation code, you can refer to https://github.com/cpystan/WSI-VQA/blob/master/downstream/show.ipynb. Given the text, we search the corresponding key word in the report and calculate the accuracy if correctly matched.
